pages/index.tsx

```diff
@@ -1,9 +1,11 @@
-// Style
 import styles from '../styles/Home.module.scss';
+import { homeSections } from '../config.yaml';
 // Internal Components
 import NavBar from '../components/NavBar';
 import EventBanner from '../components/EventBanner';
 import HeroBanner from '../components/HeroBanner';
+import Footer from '../components/Footer';
+import Section from '../layouts/Section';
```

Comment:
I think we should rename this to `HomeSection` jus tso we dont run into any naming issues between our section and the html version.